modules/adding-tab-pods-page.adoc

```diff
@@ -1,14 +1,48 @@
 // Module included in the following assemblies:
 //
-// * web_console/dynamic-plug-ins.adoc
+// * web_console/dynamic-plugin-example.adoc
 
 :_content-type: PROCEDURE
 [id="adding-tab-to-pods-page_{context}"]
 = Adding a tab to the pods page
-The following procedure adds a tab to the *Pod Details* page as an example extension to your plug-in.
+
```

Comment:
I might leave that sentence as it is good information on what you are about to do and why. It's only a single sentence and all but one of the other modules in the section have an intro sentence.